testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/BuildMultiprocessTestPlayer.cs

```diff
@@ -118,5 +119,47 @@ public static void SaveBuildInfo(BuildInfo toSave)
             var buildInfoJson = JsonUtility.ToJson(toSave);
             File.WriteAllText(Path.Combine(Application.streamingAssetsPath, BuildInfoFileName), buildInfoJson);
         }
+
+        public static bool IsMultiprocessTestPlayerAvailable()
```

Comment:
I remember seeing this in another PR, is this PR built on top of it?

Reply:
You are correct, this is part of a different PR, I reverted the change.